columbo/_interaction.py

```diff
@@ -370,7 +370,8 @@ def __init__(
         :param message: The message to be displayed to the user. If the value is callable, the argument passed in will
             be the answers that have been provided this far.
         :param options: The set of possible answers to the question. If the value is callable, the argument passed in
-            will be the answers that have been provided this far.
+            will be the answers that have been provided this far. If the value is a `Mapping`, the values of the mapping
+            will be displayed to the user.
```

Comment:
```suggestion
            will be the answers that have been provided this far. If the value is a `Mapping`, the values of the mapping
            will be displayed to the user & the respective key will be the returned value.
```
A bit more explicit about how the mapping is used.